### PR TITLE
ensure tf<2.0 for sampleUffMaskRCNN

### PR DESCRIPTION
**Pull request by rmccorm4**
Quite a few issues with this sample. I suspect pip installing TF2.0 might be a cause for some of the issues

https://github.com/NVIDIA/TensorRT/issues/132
https://github.com/NVIDIA/TensorRT/issues/125
https://github.com/NVIDIA/TensorRT/issues/123
https://github.com/NVIDIA/TensorRT/issues/159

### CODE REVIEW

**Comment by gemmit** [comment]
@rmccorm4 It was tensorflow-gpu==1.14. That didn't work for me. However, now it works with the following configuration/versions
TensorRT: 6.0.1.5
CUDA: 9.0
CuDNN: 7.6.3
python: 3.6.5
Tensorflow-gpu==1.9.0
Keras: 2.1.3

I was able to convert the model using uff-converter and run the mask-rcnn example as mentioned [here](https://github.com/NVIDIA/TensorRT/tree/master/samples/opensource/sampleUffMaskRCNN) 

Thanks

**Comment by rmccorm4** [comment]
Hm, that's strange since we were definitely pasted v1.9.0 when the samples were tested. Might be due a strange combination of things that I'm unaware of.

I'm glad it worked!
